Resources/config/collector.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="bernard.data_collector.message" class="Bernard\BernardBundle\Collector\ProducerCollector">
+        <service id="bernard.data_collector.message" class="Bernard\BernardBundle\Collector\ProducerCollector" public="true">
```

Comment:
Why does this service need to be public?

Reply:
In fact I don't know... I just took all services that where public using 3.3 and add the public tag...